plugin/src/main/shell/protoc-gen-flit

```diff
@@ -15,7 +15,7 @@
 # The quiet flag is required otherwise the tooling will dump output to stdout which will break the
 # plugin output and generation will fail.
 # -----------------------------------------------------------------------------
-DIR=$(dirname $(readlink -f "$0"))
```

Comment:
I'd suggest removing this unrelated change and PRing it upstream separately.

Reply:
will do. I was trying to get the test to pass and without changing this, it fails (assumes that it's Linux).